Codeinwp/neve
# enh: make sure search input is focused on open [HFG search icon][closes #1052]

### Summary
Make sure search input is focused when opening the HFG search icon component.


### Will affect visual aspect of the product

NO

### Test instructions


- Add the search icon component to the header
- Make sure it gets focused when opened in all three cases [canvas, minimal, floating]


Closes https://github.com/Codeinwp/neve-pro-addon/issues/1052
